### ISSUE TITLE
How does this relate to wikimedia/typescript-types 

### ISSUE THREAD

**Issue by jdlrobson** [wikimedia/mediawiki-types#3]
Does it make sense to merge efforts with https://github.com/wikimedia/typescript-types ?

**Comment by manicki** [comment]
Definitely. It has been long overdue action of mine. Not sure why but the other repository got traction, this one didn't. No point in doing the same thing twice.

This package/repository has a better name but i guess the train has departed.
I will be archiving this shortly. (happy to free the name if wanted)
